nodepool/config/templates/cluster.yaml

```diff
@@ -46,9 +46,15 @@ availabilityZone: {{.AvailabilityZone}}
 # ARN of the KMS key used to encrypt TLS assets.
 kmsKeyArn: "{{.KMSKeyARN}}"
 
-# Number of worker nodes to create
+# Number of worker nodes to create, for more control use `workerASG` and do not use this setting
 #workerCount: 1
 
+# Auto Scaling Group definition for workers. If only `workerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
+# workerASG:
+#   minSize: 1
+#   maxSize: 3
+#   rollingUpdateMinInstancesInService: 2
```

Comment:
Yes we can! I saw that and was wondering if it was the way you wanted to go, I can update it. One thing will be we have old settings outside `worker` and new settings inside. My opinion is we need to deprecate some things in order to ensure a consistent and easy to understand tool.